### DISCUSSION TITLE
Rendering style tags and components into the Shadow Dom

### DISCUSSION THREAD

**Discussion by devalirzayev** [mantinedev/mantine#5204] [Q&A] [answered]
Hi there.

I would like to know is it possible to render style tags to shadow dom instead of head?

**Comment by rtivital** [comment]
Starting from version 7, Mantine does not add any styles to head

**Reply by devalirzayev** [reply]
@rtivital, in my case seems like it does. I am using css modules for other parts of my project, so maybe because of this (webpack configuration) affects Mantine too?

<img width="400" alt="Screenshot 2023-11-07 at 10 35 28" src="https://github.com/mantinedev/mantine/assets/62056172/3f9eeec9-8e80-459a-a245-906920af03f8">

**P.S.:** I am using 7.1.7

**Reply by rtivital** [reply, marked as answer]
Usually in webpack applications CSS is loaded inline in dev mode. To extract it in production, you need to use https://webpack.js.org/plugins/mini-css-extract-plugin/

**Reply by devalirzayev** [reply]
Oh, thanks 🙌
